__tests__/utils.test.ts

```diff
@@ -455,3 +455,16 @@ describe('Test secret parsing and handling', () => {
         expect(() => parseTransformationFunction(input)).toThrow();
     });
 });
+
+describe('Version Constants', () => {
+    it('should have a valid version format', () => {
+        expect(ACTION_VERSION).toMatch(/^v\d+\.\d+\.\d+$/);
+    });
+
+    it('should return user agent with version', () => {
+        const userAgent = getUserAgent();
+        expect(userAgent).toBe(`github-action/${ACTION_VERSION}`);
```

Comment:
I would probably use a regex here too. That way you're testing behavior without coupling to implementation.